example_settings.toml

```diff
@@ -0,0 +1,34 @@
+func_name = "func_800123456"
+compiler_type = "ido"
+
+# uncomment lines below to customize the weights
+# see README.md
```

Comment:
imo for the example it's better to just include a single override here -- we don't want to maintain the commented-out weights